components/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -81,67 +71,40 @@ class D2LFilterDropdown extends mixinBehaviors([D2L.PolymerBehaviors.FilterDropd
 					// }
 				]
 			},
-			_minWidth: {
-				type: Number,
-				value: 25
-			},
-			_maxWidth: {
-				type: Number,
-				value: 400
-			},
-			_defaultSelectedTab: {
-				type: String,
-				computed: '_getDefaultSelectedTab(_filters.*)'
-			},
-			_selectedTab: {
-				type: String,
-				computed: '_getSelectedTab(_filters.*)'
-			},
 			_filterDropdownClosed: {
 				type: String,
 				value: 'd2l-filter-dropdown-closed'
 			},
-			_numFilters: {
+			_numFiltersSelectedTotal: {
```

Comment:
Super nit: I might be inclined to name this `_selectedFilterCount`.  Meh.

Reply:
I wasn't particularly happy with any of my re-naming lol, so sure!